SPECS/iscsi-initiator-utils/iscsi-initiator-utils.spec

```diff
@@ -1,244 +1,215 @@
-%global __provides_exclude_from ^(%{python2_sitearch}/.*\\.so|%{python3_sitearch}/.*\\.so)$
+%global commit0			c26218d9f8afdca44a492a4c3811648bd2880b26
 
-# Use only 7-10 first characters of the git commit hash!
-%global git_commit            2a8f9d81d0d6b5094c3fe9c686e2afb2ec27058a
-%global git_short_commit      2a8f9d8
-%global git_short_commit_date 20210729
-%global open_iscsi_build      4
-%global open_iscsi_version    2.1
+# Disable python2 build by default
+%bcond python2 0
```

Comment:
The `CGManifest` check seems to be failing for `%bcond`, can we try with `%bcond_with/%bcond_without` maybe ?